### DISCUSSION TITLE
Got Byte Order Mark in fetch response and it's prevent call response.json()

### DISCUSSION THREAD

**Discussion by Lehuyknight** [node-fetch/node-fetch#1573] [Q&A]
I got this error "message": "invalid json response body at {{url}} reason: Unexpected token ﻿? in JSON at position 0"
and i see this 
![image](https://user-images.githubusercontent.com/50477203/171530667-54dbb589-8cf1-4a63-ac1d-fd2647dfeeac.png)
the red dot is /ufeff. How can I pass that to call json() or how to convert from text() to json format. Thanks for reading and hope to find answer. I'm using node-fetch v2.6.7 and @types/node-fetch v2.6.1

**Comment by jimmywarting** [comment]
It seems to be fixed in v3 already.
https://stackblitz.com/edit/node-asvdy3?file=index.js

we have switched to using TextDecoder instead that removes BOM
